app/views/govuk_publishing_components/components/_organisation_logo.html.erb

```diff
@@ -1,21 +1,41 @@
 <%
   logo_helper = GovukPublishingComponents::Presenters::OrganisationLogoHelper.new(local_assigns)
   brand_helper = GovukPublishingComponents::AppHelpers::BrandHelper.new(organisation[:brand])
+
   organisation ||= {}
+  heading_level ||= false
+  inline ||= false
+
+  # Check if `heading_level` is a number; if so, check if it's an appropriate
+  # number.
+  use_heading = heading_level.is_a?(Integer) ?
+                  (heading_level >= 1 && heading_level <= 6) :
+                  false
+
+  # Set the wrapping element to be a heading or a `div`
+  wrapping_element = use_heading ? "h#{heading_level}" : "div"
+
+  wrapper_classes = ["gem-c-organisation-logo"]
```

Comment:
we're generally using the `%w(gem-c-organisation-logo)` notation when working with arrays of classes — I personally have no preference, but this would help with consistency.

Reply:
Okaydokey - I've changed it to use the word array shorthand.